rust/agama-cli/src/questions.rs

```diff
@@ -0,0 +1,32 @@
+use agama_lib::connection;
+use agama_lib::proxies::Questions1Proxy;
+use anyhow::{Context, Ok};
+use clap::Subcommand;
+
+#[derive(Subcommand, Debug)]
+pub enum QuestionsCommands {
+    /// To use default answers for questions.
+    UseDefaults,
+}
+
+// TODO when more commands is added, refactor and add it to agama-lib and share a bit of functionality
+async fn use_defaults() -> anyhow::Result<()> {
+    let connection = connection().await.context("Connection to DBus failed.")?;
```

Comment:
I think there is no need to add the `context` because it does not add any information to the error coming from the `connection` function.